client/js/components/map/admin/AdminLayerListItem.vue

```diff
@@ -30,119 +30,118 @@
+          :title="hintTextForLockedLayer">
+          <i :class="[iconClass,showGroupableIcon]" />
+        </a>
+      </div><!--
+   --><div class="inline-block w-1/12 text-right">
         <input
           type="checkbox"
           data-cy="adminLayerListItem:toggleDefaultVisibility"
+          :disabled="'' !== layer.attributes.visibilityGroupId || (true === isChildOfCategoryThatAppearsAsLayer)"
           @change.prevent="toggleHasDefaultVisibility"
           :checked="hasDefaultVisibility"
           :class="iconClass">
-      </div>
-      <div
-        v-if="(layer.type !== 'GisLayer' && (false === layer.attributes.layerWithChildrenHidden))"
-        class="w-2/12 text-right">
-        <!-- spacer for groups -->
-      </div>
-      <div class="w-1/12 text-right">
-        <a
-          :href="editLink"
-          data-cy="editLink">
-          <i
-            class="fa fa-pencil u-mr-0_5"
-            aria-hidden="true"
-            :title="Translator.trans('edit')" /><span class="hide-visually">{{ Translator.trans('edit') }}</span>
-        </a>
-        <button
-          class="btn--blank o-link--default u-mr-0_5"
-          data-cy="adminLayerListItem:deleteElement"
-          :title="Translator.trans('delete')"
-          @click.prevent="deleteElement"
-          v-if="childElements.length <= 0">
-          <i
-            class="fa fa-trash"
-            aria-hidden="true" /><span class="hide-visually">{{ Translator.trans('delete') }}</span>
+      </div><!--
+  --></template><!--
+          Show this Stuff for 'special category that looks like an Layer and hides all his children'
+ --><template v-if="(layer.type === 'GisLayerCategory' && layer.attributes.layerWithChildrenHidden)"><!--
+   --><div class="inline-block w-2/12 text-right">
+        <input
+          type="checkbox"
+          data-cy="adminLayerListItem:toggleDefaultVisibility"
+          @change.prevent="toggleHasDefaultVisibility"
+          :checked="hasDefaultVisibility"
+          :class="iconClass">
+      </div><!--
+     -->
+    </template><!--
+  --><div
+      v-if="(layer.type !== 'GisLayer' && (false === layer.attributes.layerWithChildrenHidden))"
+      class="inline-block w-2/12 text-right">
+      <!-- spacer for groups -->
+    </div><!--
+  --><div class="inline-block w-1/12 text-right">
+      <a
+        :href="editLink"
+        data-cy="editLink">
+        <i
+          class="fa fa-pencil u-mr-0_5"
+          aria-hidden="true"
+          :title="Translator.trans('edit')" /><span class="hide-visually">{{ Translator.trans('edit') }}</span>
+      </a>
+      <button
+        class="btn--blank o-link--default u-mr-0_5"
+        data-cy="adminLayerListItem:deleteElement"
+        :title="Translator.trans('delete')"
+        @click.prevent="deleteElement"
+        v-if="childElements.length <= 0">
+        <i
+          class="fa fa-trash"
+          aria-hidden="true" /><span class="hide-visually">{{ Translator.trans('delete') }}</span>
```

Comment:
```suggestion
        <i
          class="fa fa-trash"
          aria-hidden="true" />
          <span class="hide-visually">
            {{ Translator.trans('delete') }}
          </span>
```

Reply:
unfortunately, it breaks the layout if I change it to what you suggest. So, this component has to be refactored, as there are a lot of places that could be optimized. However, this PR just fixes an issue with the drag-and-drop function as well as the patch request. The other part od code was not changed
![image](https://github.com/user-attachments/assets/18b06b84-369f-4cdc-a7bc-a14ecf5efaab)